src/api/timetable/entity.ts

```diff
@@ -6,6 +6,14 @@ export type SemesterInfo = {
   'semester': string;
 };
 
+export type TimetableVersionInfo = {
+  id: string;
+  version: string;
+  type: string;
+  created_at: string;
+  updated_at: string;
+};
```

Comment:
timetable에서만 쓰이는 값일까요? 나중에 다른 도메인에서 이 인터페이스가 쓰일 경우는 없을까요?

Reply:
아래와 같이 type의 타입을 좁혀도 될 것 같아요.
```suggestion
export type VersionType = 'android' | 'timetable' | 'shuttle_bus_timetable' | 'express_bus_timetable' | 'city_bus_timetable';

export type TimetableVersionInfo = {
  id: string;
  version: string;
  type: VersionType ;
  created_at: string;
  updated_at: string;
};
```

Reply:
좋은 것 같습니다!! 반영했습니다!
c3c0e69840e42a5f7cc82c9afe787100f6874332